### PR TITLE
Vermont EITC

### LINKED ISSUE #3024: Vermont EITC
Reference: https://tax.vermont.gov/sites/tax/files/documents/IN-112-2022.pdf

### PR DESCRIPTION
**Pull request by Xinyi-Shao**
Fixes #3024

### Summary
📝📱🌐

Add a new policy parameter `ubi_amount` to the US tax and benefit system. Implement a basic universal basic income (UBI) reform that replaces the standard deduction and personal exemption with a flat monthly payment. Update the `reform_us.yaml` file and the `policyengine-us.ipynb` notebook to demonstrate the UBI reform.

> _`is_valid` checks_
> _refactored for clarity_
> _autumn leaves falling_

### Walkthrough
*  Add a new parameter `child_allowance.amount` to the `parameters` folder, which defines the monthly amount of the child allowance benefit per child ()

### CODE REVIEW

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: Vermont provides this rate to match the federal EITC.
+values:
+  2022-01-01: 0.38
+metadata:
+  unit: year
+  label: Vermont EITC Rate
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:6]:
```suggestion
  label: Vermont earned income tax credit match
```

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  reference:
+    title: 32 V.S.A. § 5828b Earned income tax credit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:8]:
```suggestion
    title: 32 V.S.A. § 5828b Earned income tax credit (a)
```

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    href: https://legislature.vermont.gov/statutes/section/32/151/05828b
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:9]: add 2022 and 2021 tax forms

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+description: Vermont provides this rate to match the federal EITC.
+values:
+  2022-01-01: 0.38
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:3]: add 2021 value

File: policyengine_us/variables/gov/states/vt/tax/income/vt_eitc.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class vt_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont EITC"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/vt_eitc.py:7]:
```suggestion
    label = "Vermont earned income tax credit"
```

File: policyengine_us/variables/gov/states/vt/tax/income/vt_eitc.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.vermont.gov/sites/tax/files/documents/IN-112-2022.pdf"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/vt_eitc.py:11]: add page

File: policyengine_us/variables/gov/states/vt/tax/income/vt_eitc.py [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    )
+    defined_for = StateCode.VT
+
+    def formula(tax_unit, period, parameters):
+        eitc = tax_unit("earned_income_tax_credit", period)
+        rate = parameters(period).gov.states.vt.tax.income.credits.eitc.rate
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/vt_eitc.py:17]:
```suggestion
        rate = parameters(period).gov.states.vt.tax.income.credits.eitc.rate
        return eitc * rate
```

File: policyengine_us/tests/policy/baseline/gov/states/vt/tax/income/vt_eitc.yaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+  output: 
+    mt_eitc: 380
+
+- name: EITC from federal (test 2)
+  period: 2021
+  input:
+    earned_income_tax_credit: 2_000
+    state_code: VT
+  output: 
+    mt_eitc: 760
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/vt/tax/income/vt_eitc.yaml:15]:
```suggestion
    vt_eitc: 760
```

File: policyengine_us/tests/policy/baseline/gov/states/vt/tax/income/vt_eitc.yaml [hunk 9]
```diff
@@ -0,0 +1,15 @@
+- name: EITC from federal (test 1)
+  period: 2022
+  input:
+    earned_income_tax_credit: 1_000
+    state_code: VT
+  output: 
+    mt_eitc: 380
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/vt/tax/income/vt_eitc.yaml:7]:
```suggestion
    vt_eitc: 380
```

File: policyengine_us/tests/policy/baseline/gov/states/vt/tax/income/vt_eitc.yaml [hunk 10]
```diff
@@ -0,0 +1,15 @@
+- name: EITC from federal (test 1)
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/vt/tax/income/vt_eitc.yaml:1]:
```suggestion
- name: $1000 federal EITC
```

File: policyengine_us/tests/policy/baseline/gov/states/vt/tax/income/vt_eitc.yaml [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+  period: 2022
+  input:
+    earned_income_tax_credit: 1_000
+    state_code: VT
+  output: 
+    mt_eitc: 380
+
+- name: EITC from federal (test 2)
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/vt/tax/income/vt_eitc.yaml:9]: change description + add one more test with 0 eitc

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml [hunk 12]
```diff
@@ -0,0 +1,9 @@
+description: Vermont provides this rate to match the federal EITC.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:1]:
```suggestion
description: Vermont matched the federal earned income tax credit at this rate.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:1]: changelog_entry

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml [hunk 13]
```diff
@@ -0,0 +1,9 @@
...
+values:
+  2022-01-01: 0.38
+metadata:
+  unit: year
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:5]:
```suggestion
  unit: /1
```

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml [hunk 14]
```diff
@@ -0,0 +1,14 @@
+description: Vermont matched the federal earned income tax credit at this rate.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:1]:
```suggestion
description: Vermont matches the federal earned income tax credit at this rate.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml [hunk 15]
```diff
@@ -0,0 +1,14 @@
+description: Vermont matches the federal earned income tax credit at this rate.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/credits/eitc/match.yaml:1]:
```suggestion
description: Vermont matches this fraction of the federal earned income tax credit.
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**